lib/model/EdmxModel.js

```diff
@@ -5,8 +5,8 @@ const NwCsdlSchema = require("./nw/CsdlSchema");
 const OasisCsdlSchema = require("./oasis/CsdlSchema");
 
 function getNamespace(path) {
-  let matches = /^([a-z0-9_]+)\..*/i.exec(path);
-  return matches ? matches[1] : undefined;
+  const namespacePos = path.lastIndexOf('.');
```

Comment:
> There are also other places when path is parsed and alias use is assumed.

Hi @jakub-vaclavik-sap,

I came across this set of code when I was trying to execute a CREATE operation via POST method.     
The original code did not considered Custom namespace for an odata service.     
For e.g. for `/CUS/MY_ODATA_SRV` , the `genNamespace` method returned the value **CUS**. In fact it should have returned **CUS.MY_ODATA_SRV**.

I am not versed with all LoC in this repo, so not sure where else the change might be required. I can fix them when I encounter issues in future.    
Until then can we please review the changes and merge the code.    


This is a show-stopper currently.

Reply:
"**Dots are common in subsequent path segments.**" means that there are paths like `...@Capabilities.SortRestrictions...`. for these paths your change would take whole part `...@Capabilities` as namespace and break existing functionality. There is also link to documentation where you can find this.

If this show-stopper for you, create a fork, apply your change there and use it. Write a GH issue here and we will fix it, then you can switch back to official version.

Since imported namespaces are referenced by alias, it is strange to use namespace hierarchy when there is practically always only 1 namespace but specification allows it and we should fix it.

Reply:
OK. Corrected with new fix.
Let me know if this will work.

Reply:
Actually there was still issue with default namespace (namespace omitted). Also formatting was not ok and test for the default namespace was failing. You did not see it in the PR checks, because check were not yet included in PR, after migration. It is now fixed, checks will run next time. Also the dot issue is fixed in #113. It was already implemented in EdmxModel only the occurrence in schema had to be fixed.